pkgs/development/python-modules/rectpack/default.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "rectpack";
+  version = "0.2.2";
+
+  src = fetchFromGitHub {
+    owner = "secnot";
+    repo = "rectpack";
+    rev = version;
+    hash = "sha256-kU0TT3wiudcLXrT+lYPYHYRtf7aNj/IKpnYKb/H91ng=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  # tests are base on nose
+  pythonImportsCheck = [ "rectpack" ];
+
+  meta = with lib; {
+    description = "A collection of algorithms for solving the 2D knapsack problem";
```

```suggestion
    description = "Collection of algorithms for solving the 2D knapsack problem";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

---

OK